common/djangoapps/student/management/commands/export_staff_users.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import absolute_import, print_function
+
+import logging
+from datetime import datetime, timedelta
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.core.mail.message import EmailMultiAlternatives
+from django.template.loader import get_template
+from os import remove
+
+from djqscsv import write_csv
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from student.models import CourseAccessRole
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+
+    help = """
+    This command will export a csv of all users who have logged in within the last seven days and
+    have staff access role in active courses (Courses with end date in the future).
+    """
+
+    subject = 'Staff users CSV'
+    to_addresses = ['aazam@edx.org']
+    from_address = settings.DEFAULT_FROM_EMAIL
+    txt_template_path = 'email/export_staff_users.txt'
+    html_template_path = 'email/export_staff_users.html'
+    interval = '7 days'
+
+    def handle(self, *args, **options):
+        current_date = datetime.now()
+        last_week_date = current_date - timedelta(days=7)
+        active_courses = CourseOverview.objects.filter(end__gte=current_date).values_list('id', flat=True)
+        course_access_roles = CourseAccessRole.objects.filter(
+            role__in=['staff', 'instructor'],
+            user__last_login__range=(current_date, last_week_date),
+            course_id__in=active_courses,
+            user__is_staff=False
+        ).values('user__username', 'user__email', 'role')
+        try:
+            self.send_email(course_access_roles)
+            logger.info(
+                'Sent staff users email for the period {}-{}. Staff users count:{}'.format(
+                    last_week_date,
+                    current_date,
+                    course_access_roles.count()
+                )
+            )
+        except Exception:
+            logger.info('Failed to send staff users email for the period {}-{}'.format(last_week_date, current_date))
+
+        print('Complete!')
+
+    def send_email(self, course_access_roles):
+        context = {'time_period': self.interval}
+        plain_content = self.render_template(self.txt_template_path, context)
+        html_content = self.render_template(self.html_template_path, context)
+
+        with open('staff_users.csv', 'r+') as csv_file:
```

Comment:
we are using `staff_users.csv` here and at L#68/L#65. Please have this in a variable at the class level.